### PR TITLE
[visq] Introduce 'TAE' metric error

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit introduces 'TAE' metric error.

Draft: #10432
Related to: #10393

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: compiler/visq/visq [hunk 1]
```diff
@@ -145,9 +150,9 @@ def advance_on_data(fp32_model, fq_model, data, computers):
     with tempfile.TemporaryDirectory() as fp32_dir, \
          tempfile.TemporaryDirectory() as fq_dir:
 
-        _run_dalgona(fp32_dir, data, dump_fp32_py, fp32_dir)
+        _run_dalgona(fp32_model, data, dump_fp32_py, fp32_dir)
         copyfile(fp32_dir + '/tensors.txt', fq_dir + '/tensors.txt')
-        _run_dalgona(fq_dir, data, dump_fq_py, fq_dir)
+        _run_dalgona(fq_model, data, dump_fq_py, fq_dir)
```
Comment (seanshpark) [inline compiler/visq/visq:155]: how are these changes related with TAE ?

File: compiler/visq/visq [hunk 2]
```diff
@@ -308,12 +313,13 @@ def run_on_data(fp32_model, q_model, data, dump_dot_graph, computers):
         for metric_key in computers:
             cur_computer = computers[metric_key][0]
             output = computers[metric_key][1]
+            cur_computer.advance_on(fp32_dir, fq_dir)
             if metric_key == 'MPEIR':
-                qerror_map = cur_computer.run(fp32_dir, fq_dir)
+                qerror_map = cur_computer.get_final_result()
```
Comment (seanshpark) [inline compiler/visq/visq:318]: ditto

File: compiler/visq/visq [hunk 3]
```diff
@@ -308,12 +313,13 @@ def run_on_data(fp32_model, q_model, data, dump_dot_graph, computers):
...
                 q_min = 0.0
                 q_max = 1.0
-            elif metric_key == 'MSE':
-                qerror_map, q_min, q_max = cur_computer.run(fp32_dir, fq_dir)
+            elif metric_key == 'MSE' or metric_key == 'TAE':
+                qerror_map, q_min, q_max = cur_computer.get_final_result()
```
Comment (seanshpark) [inline compiler/visq/visq:322]: ditto

File: compiler/visq/visq (hunk 1 above)
Comment (stamalakhov) [reply compiler/visq/visq:155]: @seanshpark unfortunately previous PR had some inaccuracies. So this PR fixes them and introduces `TAE` metric. Do i need to split this PR?

Comment (seanshpark) [reply compiler/visq/visq:155]: >  So this PR fixes them and introduces TAE metric. Do i need to split this PR?

I think it would be yes.

Comment (stamalakhov) [reply compiler/visq/visq:155]: ok.
